packages/monaco/src/browser/monaco-quick-input-service.ts

```diff
@@ -177,6 +179,9 @@ export class MonacoQuickInputImplementation implements IQuickInputService {
 
     private initController(): void {
         this.controller = new QuickInputController(this.getOptions());
+        this.setContextKey('inQuickOpen');
+        this.onShow(() => { this.contextKey.set(true); });
+        this.onHide(() => { this.contextKey.set(false); });
         this.updateLayout();
     }
```

Comment:
In addition to the previous comment, I think we can simplify and use the following which is present in other areas of the framework, where we register listeners in the `postConstruct`:

```diff
+ protected inQuickOpen: ContextKey<boolean>;

 @postConstruct()
 protected init(): void {
     this.initContainer();
     this.initController();
 
+    this.inQuickOpen = this.contextKeyService.createKey<boolean>('inQuickOpen', false);
+    this.onShow(() => { this.inQuickOpen.set(true); });
+    this.onHide(() => { this.inQuickOpen.set(false); })
```